src/styles/index.scss

```diff
@@ -1,3 +1,191 @@
-body {
+* {
   margin: 0;
+  padding: 0;
```

Comment:
it is redundant and bad practice to reset default margins and paddings for all selectors.
reset only for elements, which really needs it.